gurock/trcli
# Case id matchers

## Issue being resolved: https://github.com/gurock/trcli/issues/79

### Solution description
- Automated tests can be mapped to TestRail cases using the Case ID
  - On a testcase `name`
  - On a testcase `property`

More detailed info on the updated README.md file

### Changes
What changes where made?
- Added new command line option `case-matcher`
- Updated `junit_xml` reader to find case IDs in test names or properties
- Minor logging improvements
- Fixed automation_id custom field finder
- Fixed suite id not set on rollback

### Potential impacts
What could potentially be affected by the implemented changes? 
- NA

### Steps to test
- Upload results with `--case-matcher name`
- Upload results with `--case-matcher property`
- Upload results with default case matcher (should be `auto`)

### PR Tasks
